### ISSUE TITLE
Uncaught TypeError: Cannot read property '__eventMaps' of null

### ISSUE THREAD

**Issue by hems** [Tarang/Meteor-Analytics#27]
![screen shot 2015-06-03 at 11 01 10](https://cloud.githubusercontent.com/assets/27327/7957523/eaf2d9c8-09df-11e5-9b7f-296577962a3b.png)

my app is super vanilla at the moment, perhaps you expecting maps for something i didnt not use events?

**Comment by Tarang** [comment]
I thought we fixed this. Could you give me the output of `meteor --list`

**Comment by hems** [comment]
`$ meteor list`

```
autoupdate               1.2.0* Update the client when new client code is ava...
coffeescript             1.0.6  Javascript dialect with fewer braces and semi...
copleykj:livestamp       1.1.7  jQuery plugin providing auto-updating timeago...
fastclick                1.0.3  Faster touch events on mobile
mdg:geolocation          1.0.3  Provides reactive geolocation on desktop and ...
meteor-platform          1.2.2  Include a standard set of Meteor packages in ...
mquandalle:stylus        1.1.1  Expressive, dynamic, robust CSS
mrt:nprogress            0.2.0  NProgress for Meteor
mrt:redis                0.1.3  Meteor package to wrap redis driver. Includes...
mystor:device-detection  0.2.0  Client-Side Device Type Detection & Template ...
outatime:geolib          2.0.15  Geolib - Library to perform geo specific tasks
raix:eventemitter        0.1.2  A server and client event package.
tail:core                0.9.5* An analytics package for meteor


* New versions of these packages are available! Run 'meteor update' to try to
  update those packages to their latest versions. If your packages cannot be
  updated further, try typing `meteor add <package>@<newVersion>` to see more
  information.

```

**Comment by Tarang** [comment]
@hems Could you include one with the analytics package? The previous version had this problem just checking if i published it correctly.

**Comment by hems** [comment]
@Tarang sorry by accident i posted without it! 

i just added it back and updated my reply!

**Comment by hems** [comment]
for some reason it is on 0.9.5!  i'll do an update and see where it goes...

**Comment by hems** [comment]
meteor update fixed the issue!

sorry about that! ( it's weird i had the old version, i think i probably installed before you released 0.9.6 ) !
